ui/apps/platform/src/Containers/DeferralConfiguration/VulnerabilitiesConfiguration.tsx

```diff
@@ -19,41 +23,53 @@ import {
     Title,
 } from '@patternfly/react-core';
 import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import get from 'lodash/get';
 import isEqual from 'lodash/isEqual';
 import sortBy from 'lodash/sortBy';
 import { FormikHandlers, useFormik } from 'formik';
+import * as yup from 'yup';
 
 import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
 import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
 import { VulnerabilitiesDeferralConfig } from 'services/DeferralConfigService';
+import useToasts, { Toast } from 'hooks/patternfly/useToasts';
+
+import usePermissions from 'hooks/usePermissions';
```

Comment:
I was wondering how we're ordering imports since there is a break here. Usually, I try to separate our internal imports by utils/hooks/modules and components